Main/ClientApp/src/Components/SpotifyCallbackPage.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Redirect } from "react-router-dom";
+import { UrlHash } from "../Infrastructure/UrlUtilities";
+import Cookies from "js-cookie";
+import { useCookie } from "../Infrastructure/CustomReactHooks";
+
+export default function SpotifyCallbackPage() {
+  const hash = window.location.hash;
+  let params = UrlHash(hash);
+
+  Cookies.set("spotifyUserToken", JSON.stringify(params.get("access_token")));
+  let [collectionId, _] = useCookie<number | null>(
```

Comment:
It would be better to use Cookies.get here, there is not actuall state so no need to use a hook